velox/functions/lib/Re2Functions.cpp

```diff
@@ -391,6 +391,49 @@ bool matchExactPattern(
       std::memcmp(input.data(), pattern.data(), length) == 0;
 }
 
+// Match the input(from the position of start) with relaxed pattern(from the
+// specified subPattern).
+bool matchRelaxedFixedPattern(
+    StringView input,
+    const std::vector<SubPatternMetadata>& subPatterns,
+    size_t length,
```

Comment:
nit: perhaps, change the order of arguments to start, length (seems more conventional)